.github/workflows/my-ci-pipeline.yml

```
@@ -0,0 +1,33 @@
+name: Continuous Integration 
+on:
+  push:
+    branches: [main]
+    paths-ignore:
+      - 'README.md'
+  pull_request:
+    branches: [main]
+    paths-ignore:
+      - 'README.md'    
+
+jobs: 
+ build: 
+  name: Build and test 
+  runs-on: ubuntu-latest 
+  steps: 
+   - uses: actions/checkout@v2 
+   - run: docker build --target test --tag todo-app:test .
+   - run: docker run -e TRELLO_KEY={secrets.TRELLO_KEY} -e TRELLO_TOKEN={secrets.TRELLO_TOKEN} -e TRELLO_BOARD_ID={secrets.TRELLO_BOARD_ID} todo-app:test
```

Is there a reason you're passing your secrets through here, even though your tests (I _believe_) only rely on your fake test variables? E.g. we could instead use:
```suggestion
   - run: docker run --env-file .env.test todo-app:test
```
And then we'd save ourselves from worrying about secrets here at all